spec/views/tutoring_sessions/index_spec.rb

```diff
@@ -3,15 +3,19 @@
 RSpec.describe 'tutoring_sessions/index', type: :view do
   before(:each) do
     @tutoring_session = create(:tutoring_session, location: 'library',
-                                                  start_time: '2017-03-03',
-                                                  end_time: '2017-03-03')
+                                                  hours: '3',
+                                                  start_date: '2017-01-01',
+                                                  end_date: '2017-12-13',
+                                                  session_comment: 'hi')
```

Comment:
We've been doing that since the beginning. https://github.com/LiteracyVolunteersOfMA/lvm-rails/blob/master/spec/views/student/index_spec.rb#L6
I used `Faker::Number` to generate the hours, rails give a TypeError because it expects a regex not an int.
It's easier to just use `'3'`.